assets/sass/cds/_sections.scss

```diff
@@ -949,7 +949,7 @@ span.livefr {
   flex-direction: row;
   flex-wrap: wrap;
 
-  @include mobile_only {
+  @media (max-width: 950px) {
```

Comment:
Should we maybe consider adding this to our variables, since it's referenced in multiple places?